DependencyInjection/AzineEmailUpdateConfirmationExtension.php

```diff
@@ -29,6 +29,9 @@ public function load(array $configs, ContainerBuilder $container)
             $container->setParameter('azine_email_update_confirmation.template', $config['email_template']);
             $container->setParameter('azine_email_update_confirmation.cypher_method', $config['cypher_method']);
             $container->setParameter('azine_email_update_confirmation.redirect_route', $config['redirect_route']);
+            $container->setParameter('azine_email_update_confirmation.from_email', $config['from_email']);
```

Comment:
is it possible to use the 'fos_user_from_email_address" parameter as default? See https://symfony.com/doc/master/bundles/FOSUserBundle/index.html#step-5-configure-the-fosuserbundle